protocols/gossipsub/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.45.0 (unreleased)
+
+- Signed messages use sequential integers in the sequence number field. See [PR
+	3551].
```

Comment:
```suggestion
# 0.41.1 - unreleased

- Signed messages now use sequential integers in the sequence number field.
  See [PR 3551].
```

Reply:
I can't seem to accept my own suggestion, likely because this PR is from an org-owned fork so I don't automatically get write permissions.